docs/sources/enterprise/reporting.md

```diff
@@ -55,18 +90,37 @@ The last saved version of the report will be sent to selected emails. You can us
 
 {{< docs-imagebox img="/img/docs/enterprise/reports_send_test_mail.png" max-width="500px" class="docs-image--no-shadow" >}}
 
+## Send report via the API
+
+You can send reports programmatically with the [send report]({{< relref "../http_api/reporting.md#send-report" >}}) endpoint in the [HTTP APIs]({{< relref "../http_api" >}}).
+
 ## Rendering configuration
 
-When Grafana generates a report, it will render each panel separately and then put them together in a PDF file. You can configure the per-panel rendering request timeout and the maximum number of concurrent calls to the rendering service. These options are available in the [configuration]({{< relref "../installation/configuration.md">}}) file.
+When generating reports, each panel renders separately before being collected in a PDF. The per panel rendering timeout and number of concurrently rendered panels can be configured.
+
+To modify the panels' clarity you can set a scale factor for the rendered images. A higher scale factor is more legible but will increase the file size of the generated PDF.
+
+ These options are available in the [configuration]({{< relref "../installation/configuration.md">}}) file.
 
 ```ini
 [reporting]
 # Set timeout for each panel rendering request
 rendering_timeout = 10s
 # Set maximum number of concurrent calls to the rendering service
 concurrent_render_limit = 4
+# Set the scale factor for rendering images. 2 is enough for monitor resolutions
+# 4 would be better for printed material. Setting a higher value affects performance and memory
+image_scale_factor = 2
 ```
 
+## Report time range
+
+Reports uses the saved time range of the dashboard. Changing the time range of the report is done by saving a modified time range to the dashboard.
+
+The page header of the report displays the time range for the dashboard's data queries. Dashboards set to use the browser's time zone will use the time zone on the Grafana server.
+
+If the time zone is set differently between your Grafana server and its remote image renderer, then the time ranges in the report may be different between the page header and the time axes in the panels. We advise always setting the time zone to UTC for dashboards when using a remote renderer to avoid this. 
```

Comment:
```suggestion
If the time zone is set differently between your Grafana server and its remote image renderer, then the time ranges in the report might be different between the page header and the time axes in the panels. We advise always setting the time zone to UTC for dashboards when using a remote renderer to avoid this. 
```

Reply:
Use "might" instead of "may" when talking about possibilities. "May" has connotations of permission. "Yes, you may do that" vs. "That might happen."